gradle/scripts/lib/java-alpn.gradle

```diff
@@ -1,5 +1,5 @@
 // JDK 9 does not require Jetty ALPN agent at all.
-if (JavaVersion.current() >= JavaVersion.VERSION_1_9) {
+if (JavaVersion.current() >= JavaVersion.VERSION_1_9 && System.env.TEST_JAVA_VERSION != '8') {
```

Comment:
I have updated `gradle-scripts` for you. You'll not need this anymore once you rebase.
https://github.com/line/gradle-scripts/commit/a0a57120fb832fe93d23ce335d7316e99b756077